docs/ruleset_functions/is_zone_mechanically_cooled.md

```diff
@@ -0,0 +1,30 @@
+# is_zone_mechanically_cooled
+**Schema Version:** 0.0.22  
+
+**Description:** determines whether a zone is mechanically cooled or not.  DOES NOT CHECK FOR TRANSFER AIR
+
+**Inputs:**
+- **RMD**
+- **zone**
+
+**Returns:**  
+- **result**: a boolean - TRUE or FALSE
+ 
+**Function Call:**
+- **get_list_hvac_systems_associated_with_zone**
+
+## Logic:
+- set result to FALSE: `result = FALSE`
+- get the hvac systems serving the zone by using the get_list_hvac_systems_associated_with_zone function: `list_hvac_systems = get_list_hvac_systems_associated_with_zone(RMD,zone)`
```

Comment:
Is this a function that hasn't been written?

I am not sure how many times this function will be called, but we can also write here as 

for terminal in zone.terminals:
    list_hvac_system_ids.append(terminal.served_by_heating_ventilation_air_condioining_system).

A list of hvac ids should give a good starting point to continue the process below.

Reply:
this is a function that exists already